### PR TITLE
Issues/#3 Enables GPU mining

### PR DESCRIPTION
**Pull request by macterra**

### CODE REVIEW

File: opencl/keccak.cl [hunk 1]
```diff
@@ -105,7 +105,7 @@ __kernel void search(
 	state7 = in[7];
 	state8 = in[8];
 	state9 = (uint2)(in[9].x, base + get_global_id(0));
-	state10 = (uint2)(1,0);
+	state10 = (uint2)(6,0);
```
Comment (macterra) [inline opencl/keccak.cl]: This converts the OpenCL Keccak to SHA-3

File: winhacks.h [hunk 2]
```diff
@@ -17,3 +17,6 @@
 
 #include <winerror.h>
 
+#ifdef interface
+#undef interface
+#endif
```
Comment (macterra) [inline winhacks.h]: This was necessary to build for mingw when configured with --disable-other-drivers

File: miner.c [hunk 3]
```diff
@@ -10650,7 +10650,7 @@ void work_hash(struct work * const work)
 	const struct mining_algorithm * const malgo = work_mining_algorithm(work);
 	malgo->hash_data_f(work->hash, work->data);
 
-    applog(LOG_DEBUG, ">> work_hash malgo = %s", malgo->name);
+    // applog(LOG_DEBUG, ">> work_hash malgo = %s", malgo->name);
```
Comment (r-soltani) [inline miner.c]: Why are the comments disabled?

File: opencl/keccak.cl (hunk 1 above)
Comment (r-soltani) [reply opencl/keccak.cl]: Is this the only change in regards to coverting Keccak to SHA3?

**Review by ealefad — APPROVED** [review]

File: opencl/keccak.cl (hunk 1 above)
Comment (macterra) [reply opencl/keccak.cl]: yes

File: miner.c (hunk 3 above)
Comment (macterra) [reply miner.c]: I added them for debugging.

**Pull request merged**